bin/meteor.sh

```diff
@@ -17,8 +17,10 @@ get_meteor_minor_version() {
 # Format of .meteor/release file is METEOR@1.4.x-patchsomething
 meteor_node_version() {
   minor=$(get_meteor_minor_version)
-  if [ "$minor" -ge 9 ] ; then
-    echo "13.7.x"
+  if [ "$minor" -ge 10 ] ; then
+    echo "12.16.x"
```

Comment:
Why this special case ? This will fall in the if minor > 9

Reply:
This is to explicitly add support for Meteor 1.10.x in case they change the Node.js version they want for this specific version. But maybe it's useless.

This is what has been done for Meteor 1.8 and 1.6.